lib/container/container.go

```diff
@@ -129,3 +132,13 @@ func WaitContainer(id string) (int, error) {
 
 	return client.WaitContainer(id)
 }
+
+func getDockerHost() (host string) {
```

Comment:
> I'll make another pull request to fix this then.

:+1: awesome.

@peter-edge you had an example of some code that did deal with tls-enabled daemons right?